admin/listPurchases/listView.html

```diff
@@ -107,7 +107,7 @@
             <td>${entry.customer || 'En kunde'}</td>
             <td class="status">${translateStatus(entry.status)}<br><span class="status-date">${entry.statusChangeDate || "?"}</span></td>
             <td>${badgeImages.map((imageUrl) => `<img class="image-thumbnail" src="${imageBaseUrl + imageUrl}">`).join('')}</td>
-            <td>${entry.isPreorder ? 'PRE: ' : ''}${entry.description}</td>
+            <td>${entry.isPreorder ? 'PRE: ' : ''}${entry.description.replaceAll(" + ", "<br>").replaceAll(",",":")}</td>
```

Comment:
I would prefer to parse this from some data somewhere... what other info does entry have? Can we render out e.g. `entry.items.map((item) => `${item.name}, ${item.quantity} stk.`).join("<br>")` or something like that?

I'd like to be able to change `entry.description` to whatever we think is easiest to read at any point in time - without breaking code that tries to parse that text string and use it for other things.

Reply:
entries didn't have a field for items, but they do now!